DDDEastAnglia.Tests/SponsorBuilder.cs

```diff
@@ -0,0 +1,54 @@
+using System;
+using DDDEastAnglia.DataAccess.SimpleData.Models;
+
+namespace DDDEastAnglia.Tests
+{
+    internal class SponsorBuilder
+    {
+        private byte[] logo;
+        private string name;
+        private DateTime? paymentDate;
+        private bool showPublicly = true;
+        private int sponsorId;
+        private int sponsorshipAmount;
+        private string url;
+
+        public Sponsor Build()
```

Comment:
There are two things here:

1. The amount sponsored is primarily (but not exclusively) a function of the sponsorship level (Platinum/Gold/Silver/Bronze)
2. We *always* talk about sponsorship levels rather than amounts, so the code is modelling an implementation detail of the domain, rather than expressing the concept in domain terminology.